templates/emails/virtualcard.purchase.hbs

```diff
@@ -0,0 +1,26 @@
+Subject: Virtual Card Purchase
+
+{{> header}}
+
+<center>
+  <h1>
+    A card attached to {{collective.name}} was charged {{currency expense.amount currency=expense.currency
+  precision=2}}.
+  </h1>
+</center>
+<p>
+A card attached to {{collective.name}} was charged {{currency expense.amount currency=expense.currency
+  precision=2}}. The responsible admin {{user.firstName}} {{user.lastName}} will be asked to submit additional
```

Comment:
firstName/lastName is deprecated in favor or `name` on the `Collective` model. It's actually not part of the `User` model anymore I believe?

Reply:
That's correct. I've refactored this part to use the `Collective` model.